README.rst

```diff
@@ -208,11 +217,11 @@ ansible-lint on them works:
 
     [ANSIBLE0002] Trailing whitespace
     examples/example.yml:13
-        action: do nothing   
+        action: do nothing
```

Comment:
@awcrosby this edit is wrong: it demonstrates a trailing whitespace error.

Reply:
thanks! I will correct